config/sysinfo/commands

```diff
@@ -0,0 +1,15 @@
+df -mP
+dmesg
+uname -a
+lspci -vvnn
+gcc --version
+ld --version
+hostname
+uptime
+dmidecode
+ifconfig -a
+brctl show
+ip link
+numactl --hardware show
```

Comment:
what is the current result if the dependency package is not installed in system for the commands?

Reply:
@sathnaga They are simply logged in the job.log like this:
```
2020-05-21 07:52:24,346 sysinfo          L0173 DEBUG| Not logging 'dmidecode' (command 'dmidecode' was not found)
2020-05-21 07:52:30,906 sysinfo          L0105 DEBUG| Not logging /proc/pci (file does not exist)
2020-05-21 07:52:30,934 sysinfo          L0173 DEBUG| Not logging 'brctl show' (command 'brctl' was not found)
2020-05-21 07:52:31,202 sysinfo          L0173 DEBUG| Not logging 'dmidecode' (command 'dmidecode' was not found)
2020-05-21 07:52:40,739 sysinfo          L0105 DEBUG| Not logging /proc/pci (file does not exist)
2020-05-21 07:52:40,763 sysinfo          L0173 DEBUG| Not logging 'brctl show' (command 'brctl' was not found)
```

Reply:
Not harmful...

Reply:
Definitely not harmful